### DISCUSSION TITLE
Updating metadata to multiple files

### DISCUSSION THREAD

**Discussion by Lmasey** [mdelobelle/metadatamenu#624] [General]
I'm not sure if I'm going to word this right, but is this something I would be able to do in metadata menu without creating 100s of notes each day:)

I have 100+ students, I assess them over multiple targets each day, I do not want the new entry to change the old entry. 

I know I can create a file class for student and a file class for the assessment data, the problem is each day I would have to create a new assessment file for each student! 

I guess what I'm asking is there a way to have the student names on the right (Each file name) and then add to the table each data point based on the metadata menu fields? Things like score, growth from last entry (would love if this somehow auto populated) chance of mastery on the state test etc. 

I know there is multi select, but will that put each name in the same block? I need each name to correspond to its own data set. Should I make a field for each file name? 

Also, can we make charts from metadata tables?

**Comment by mdelobelle** [comment]
You may have a look at ObjectList field type
in the Student fileclass, create a "assessments" field of type "ObjectList"
Then create fields with "assessments" as parent field:
- "assessment date"
- "score"
- "growth"
- "chance of mastery"
- ....
Your data points for a given student will be available in the frontmatter as a list

**Reply by Lmasey** [reply]
Where I'm stumped is, do I need to create a new note for each student per test? We do 3-5 daily assessments per week. I can't wrap my head around how to do Johnny made this and has this data but Susie scored this without duplicating the note for each child.
